astroquery/sdss/core.py

```diff
@@ -55,19 +56,21 @@ class SDSSClass(BaseQuery):
                            'qso_bal': [30, 31], 'qso_bright': 32
                            }
 
-    def query_crossid_async(self, coordinates, obj_names=None,
-                            photoobj_fields=None, specobj_fields=None,
-                            get_query_payload=False, timeout=TIMEOUT,
-                            radius=5. * u.arcsec,
+    def query_crossid_async(self, coordinates, radius=5. * u.arcsec, timeout=TIMEOUT,
+                            fields=None, photoobj_fields=None, specobj_fields=None, obj_names=None,
+                            spectro=False, region=False, field_help=False, get_query_payload=False,
```

Comment:
OK, I can rearrange.

Reply:
please do not rearrange, rather make them mandatory kwarg. We incrementally doing this change in other modules, and examples like these are excellent cases where optional parameters should be forced to spelled out. 
(Of course it will be an API break, but I plan to have multiple API breaks in the next release anyway by doing significant cleanups).

Reply:
Is there a good example of that for me to look at?

Reply:
Yeap, here is the PEP: https://peps.python.org/pep-3102/

For astroquery e.g. alma, esasky, mast are all moved on this approach.

In practice you add a `*` between positional and keyword arguments, and then can rearrange the order of the keyword ones as you wish (I fully agree that in many cases it makes sense to rearrange them, but if we don't enforce them to be keyword only, any addition is a breaking API change.)

Reply:
I think using keyword-only arguments is a good idea, but it might be better to apply that change to the entire `sdss` subpackage at once instead of doing it one function at a time.